app/api/healthcareAuthorities/getHealthcareAuthoritiesApi.ts

```diff
@@ -25,7 +25,8 @@ const getHealthcareAuthoritiesApi = async (
   yamlUrl: string = AUTHORITIES_YAML_ROUTE,
 ): Promise<HealthcareAuthority[]> => {
   const yamlString = await fetch(yamlUrl).then((res) => res.text());
-  let { authorities } = Yaml.safeLoad(yamlString);
+  const record = Yaml.safeLoad(yamlString) as Record<string, any>;
```

Comment:
just a note, we really want to be avoiding type casting, as its basically removing all of the benefit of ts. Is there away we can express this without the `as`?

Reply:
`const record: Record<string, any> = Yaml.safeLoad(yamlString)` <- is this any better? Also unsure if there's a way to get around using `any` as the record value type here (`[String: Any]` is often how I unpack JSON in swift, but using `Any` always feels a little suspect) 🤔

Reply:
yup! the `const thing: Type = foo` syntax will fail tsc if foo is not of the type of Type as were `const thing = foo as Type` will not.

anys are also good to avoid, looks like something like `Record<string, string>` might work?

Reply:
ah, I think we might be stuck with `any`; there are a few different types in there. But good to know - thank you for unpacking things conceptually for me!